region/tourian/main/Metroid Room 1.json

```diff
@@ -643,134 +694,114 @@
-        "Group all of the Metroids by hitting the first Rinka with a Power Bomb.",
-        "Once grouped, use two more Power Bombs to finish them off."
-      ]
-    },
-    {
-      "link": [2, 4],
-      "name": "Ice Evade",
+      "link": [3, 2],
+      "name": "Space Jump",
       "requires": [
-        "Ice"
+        "SpaceJump",
+        {"or": [
+          {"obstaclesCleared": ["A"]},
+          "Ice",
+          "ScrewAttack",
+          "canPseudoScrew",
+          "canMetroidAvoid",
+          {"metroidFrames": 270}
+        ]}
       ]
     },
     {
-      "link": [2, 4],
-      "name": "Screw Evade",
+      "link": [3, 2],
+      "name": "Speedjump",
       "requires": [
+        "SpeedBooster",
+        "canCarefulJump",
         {"or": [
+          {"obstaclesCleared": ["A"]},
+          "Ice",
           "ScrewAttack",
-          "canPseudoScrew"
-        ]},
-        {"or": [
-          "canDodgeWhileShooting",
-          "SpaceJump"
+          "canPseudoScrew",
+          "canMetroidAvoid",
+          {"metroidFrames": 130}
         ]}
       ],
-      "note": [
-        "Use ScrewAttack or a PseudoScrew to prevent Metroids from attaching to Samus.",
-        "These abilities may also be used to temporarily prevent damage from Metroids if they do attach."
-      ],
-      "devNote": "FIXME: Screw effects cannot be used during some of the strats used in crossing the acid pit."
-    },
-    {
-      "link": [2, 4],
-      "name": "Harder Evade",
-      "requires": [
-        "canMetroidAvoid"
-      ]
+      "note": "Use the full runway to gain echoes and enough speed to reach the far platform."
     },
     {
-      "link": [2, 4],
-      "name": "Blue SpaceJump",
+      "link": [3, 2],
+      "name": "Shinespark Midair",
       "requires": [
-        "canBlueSpaceJump",
+        "canMidairShinespark",
         {"canShineCharge": {
-          "usedTiles": 22,
-          "openEnd": 1
+          "usedTiles": 31,
+          "openEnd": 0
         }},
-        {"metroidFrames": 4}
+        {"or": [
+          {"and": [
+            {"obstaclesCleared": ["A"]},
+            {"shinespark": {
+              "frames": 54
+            }}
+          ]},
+          {"and": [
+            {"metroidFrames": 300},
+            {"shinespark": {
+              "frames": 54
+            }}
+          ]},
+          {"and": [
+            "canTrickyJump",
+            {"shinespark": {
+              "frames": 75,
+              "excessFrames": 16
+            }}
+          ]}
+        ]}
       ]
     },
     {
-      "link": [2, 4],
-      "name": "Blue SpaceJump, Into the Room",
-      "entranceCondition": {
-        "comeInShinecharging": {
-          "length": 1,
-          "openEnd": 1
-        }
-      },
+      "link": [3, 2],
+      "name": "Metroid Room 1 Spring Ball Jump (Left to Right)",
       "requires": [
-        "canBlueSpaceJump",
-        "canCarefulJump"
+        "h_canUseSpringBall",
+        "canLateralMidAirMorph",
```

Comment:
Is this needed?
```suggestion
        "canSpringBallBounce",
```

Reply:
`canSpringBallBounce` has been underdefined in how it should be used.  I like to think it is for when you need to control the height of your jumps.
I think right now it means any kind of bouncing, but I can see it being changed through an issue later.